flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java

```diff
@@ -58,10 +65,57 @@ public AndroidModuleLibraryManager(@NotNull Project project) {
     super(project);
   }
 
-  public void update() {
+  public void updateOld() {
```

Comment:
can you delete this method or give it a clearer name?

Reply:
Done, along with substantial additional changes. PTAL